e2e/external_dep/WORKSPACE

```diff
@@ -5,8 +5,9 @@ local_repository(
     path = "../..",
 )
 
-load("@aspect_rules_ts//ts:repositories.bzl", "LATEST_VERSION", "rules_ts_dependencies")
-rules_ts_dependencies(ts_version = LATEST_VERSION)
+load("@aspect_rules_ts//ts:repositories.bzl", "LATEST_TS_VERSION", "rules_ts_dependencies")
+
+rules_ts_dependencies(ts_version = LATEST_TS_VERSION)
```

Comment:
hmmm. how about LATEST_TYPESCRIPT_VERSION to match the npm package name of the tool?

Reply:
Yeah I like that idea, can do the same with rollup etc.

Updated.